spec/rubocop/comment_config_spec.rb

```diff
@@ -50,7 +50,7 @@
         '# rubocop:disable Layout/BlockAlignment some comment why',
         '# rubocop:enable Style/Send, Layout/BlockAlignment but why?',
         '# rubocop:enable Lint/RandOne foo bar!',            # 43
-        '# rubocop:disable Lint/EmptyInterpolation',
```

Comment:
It was a bug in test.
Example description for this cop reads "supports enabling/disabling cops without a prefix", but cop was WITH a prefix :)

Reply:
Nice catch!